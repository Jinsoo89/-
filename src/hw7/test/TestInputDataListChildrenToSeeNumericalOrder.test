# Test the list children with data without parsing from exist data

# Create graph
CreateGraph players

# Add nodes and edges
AddNode players C.Ronaldo
AddNode players D.Maria
AddNode players L.Messi
AddEdge players C.Ronaldo D.Maria 1
AddEdge players C.Ronaldo D.Maria 7
AddEdge players C.Ronaldo L.Messi 4
AddEdge players C.Ronaldo D.Maria 9
AddEdge players C.Ronaldo L.Messi 1.5
AddEdge players C.Ronaldo L.Messi 1.5
AddEdge players C.Ronaldo L.Messi 0.1

# list children nodes in the graph
ListChildren players C.Ronaldo