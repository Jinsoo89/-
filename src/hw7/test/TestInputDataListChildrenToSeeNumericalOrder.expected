# Test the list children with data without parsing from exist data

# Create graph
created graph players

# Add nodes and edges
added node C.Ronaldo to players
added node D.Maria to players
added node L.Messi to players
added edge 1.000 from C.Ronaldo to D.Maria in players
added edge 7.000 from C.Ronaldo to D.Maria in players
added edge 4.000 from C.Ronaldo to L.Messi in players
added edge 9.000 from C.Ronaldo to D.Maria in players
added edge 1.500 from C.Ronaldo to L.Messi in players
added edge 1.500 from C.Ronaldo to L.Messi in players
added edge 0.100 from C.Ronaldo to L.Messi in players

# list children nodes in the graph
the children of C.Ronaldo in players are: D.Maria(1.000) D.Maria(7.000) D.Maria(9.000) L.Messi(0.100) L.Messi(1.500) L.Messi(4.000)